Source/JavaScriptCore/dfg/DFGAbstractInterpreterInlines.h

```diff
@@ -4330,6 +4334,7 @@ bool AbstractInterpreter<AbstractStateType>::executeEffects(unsigned clobberLimi
             case Array::Int32:
             case Array::Double:
             case Array::Contiguous:
+            case Array::AlwaysSlowPutContiguous:
```

Comment:
do we have tests for `for in`?

Reply:
Added `JSTests/stress/for-in-always-slow-put-contiguous.js`.